resources_pages/install_ds_stack_mac.md

```diff
@@ -153,34 +153,40 @@ you should see something like this (does not have to be the exact same version)
 git version 2.24.2 (Apple Git-127)
 ```
 
-> Note: If you run into trouble, please see that Install Git > Mac OS section from [Happy Git and GitHub for the useR](http://happygitwithr.com/install-git.html#mac-os) for additional help or strategies for Git installation.
+> **Note:** If you run into trouble, please see that Install Git > Mac OS section from [Happy Git and GitHub for the useR](http://happygitwithr.com/install-git.html#mac-os) for additional help or strategies for Git installation.
 
 ### Configuring Git user info
 
 Next, we need to configure Git by telling it your name and email. To do this type the following into the terminal (replacing Jane Doe and janedoe@example.com, with your name and email (the same used to sign up for GitHub), respectively):
 
-```
+```bash
 git config --global user.name "Jane Doe"
 git config --global user.email janedoe@example.com
 ```
 
-> Note: to ensure that you haven't made a typo in any of the above, you can view your global Git configurations by either opening the configuration file in a text editor (e.g. via the command `code ~/.gitconfig`) or by typing `git config --list --global`.
+> **Note:** to ensure that you haven't made a typo in any of the above, you can view your global Git configurations by either opening the configuration file in a text editor (e.g. via the command `code ~/.gitconfig`) or by typing `git config --list --global`.
 
 ### Setting VS Code as the default editor
 
-To make programs run from the terminal (such as `git`) use vscode by default, we will modify `~/.bash_profile`. First, open it using VS Code:
+To make programs run from the terminal (such as `git`) use vscode by default, we will modify `~/.bash_profile`. First, maopen it using VS Code:
 
-```
+```bash
 code ~/.bash_profile
 ```
 
-> Note: If you see any existing lines in your `~/.bash_profile`
+> **Note**: if the file `~/.bash_profile` does not already exist, make one using the following command:
+>
+> ```bash
+> touch ~/.bash_profile
```

Comment:
The file should automatically be created when you run the `code` command above. Could you try with a random non-existing file name and confirm that the file is created when you save in `vscode` without having to `touch` it first??

Reply:
You're right. `code ~/.bash_profile` creates the file if it doesn't exist, but only if we save it right there. I think it's still helpful to remind the students that the file might not be already there.

Reply:
Good idea, I added a parenthesized suggestion above. Do you think that is enough and we can delete this section?

Reply:
Thanks, yes I think we can delete this note box.